.travis.yml

```diff
@@ -16,7 +16,7 @@ install:
 - pip install --upgrade --force-reinstall "setuptools; python_version != '3.2' and python_version != '3.3'" "setuptools < 30; python_version == '3.2'" "setuptools < 40; python_version == '3.3'"
 - pip uninstall --yes six || true
 - pip install --upgrade --force-reinstall --ignore-installed -e .
-- pip install pytest
+- pip install pytest typing
```

Comment:
Is it possible to test this without depending on typing?

Reply:
Sure, this was just the simplest way. I thought it is OK to add it here since `typing` will not be a dependency for `six` itself. Do you want me to add an extra test, or replace the existing one? (I would prefer to keep the existing one.)

Reply:
(FWIW I added a typing-agnostic test.)

Reply:
Yes, I mainly wanted to make sure that the semantic change was tested directly rather than relying on `typing`.